core/testMinimumPhpVersion.php

```diff
@@ -84,6 +84,12 @@
      */
     function Piwik_ShouldPrintBackTraceWithMessage()
     {
+        if (\Piwik\SettingsServer::isArchivePhpTriggered()
```

Comment:
Not sure if backtrace should be included when triggered through the web? Would maybe rather not do it.